src/main/java/wolox/training/repositories/UserRepository.java

```diff
@@ -10,4 +12,6 @@
 
     Optional<User> findFirstByUsername(String username);
 
+    Optional<List<User>> findByBirthdateBetweenAndNameIgnoreCaseContaining(LocalDate start, LocalDate end,
```

Comment:
When a list is expected to be returned you shouldn't use the Optional, since when a List has no data it's not null, it's empty. 😁

Reply:
Done :D